test/known_issues/test-path-parse.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const path = require('path');
+
+// Issue: https://github.com/nodejs/node/issues/6229
```

Comment:
Could you move this to the top, just under `'use strict;'`, and change `Issue` to `Refs` for consistency with the other known issue tests.